src/app/genes/gene-similar/gene-similar.component.ts

```diff
@@ -201,17 +199,34 @@ export class GeneSimilarComponent implements OnInit {
             this.apiService.getInfosMatchIds(nodesIds).subscribe((datas: GeneInfosResponse) => {
                 this.genesInfo = datas.items;
                 this.genesInfo.forEach((de: GeneInfo) => {
-                    // First map all entries nested in the data to a new array
-                    const meArray = (de.medianexpression.length) ? de.medianexpression.map(
-                        (me: MedianExpression) => me.tissue) : [];
 
-                    // Join the final strings into a new array removing duplicates
-                    de.brain_regions = (meArray.length) ? meArray.filter(this.getUnique)
-                        .sort((a: string, b: string) => a.localeCompare(b)).join(', ') : '';
+                    // Populate display fields
+                    de.isChangedInADBrain = (de.isChangedInADBrain) ? de.isChangedInADBrain : false;
+                    de.nominated_target_display_value = de.nominations > 0;
 
-                    de.num_brain_regions = meArray.length.toString();
-                    de.isChangedInADBrain = (de.isChangedInADBrain) ? de.isChangedInADBrain :
-                        false;
+                    // Populate MedianExpression display fields
+                    const meArray = (de.medianexpression.length) ?
+                        de.medianexpression.map((me: MedianExpression) => me.tissue) : [];
+                    de.brain_regions_display_value = (meArray.length) ? meArray.filter(this.getUnique)
+                        .sort((a: string, b: string) => a.localeCompare(b)).join(', ') : '';
+                    de.num_brain_regions_display_value = meArray.length.toString();
+
+                    // Populate Druggability display fields
+                    if (de.druggability && de.druggability.length) {
+                        de.pharos_class_display_value = de.druggability[0].pharos_class ?
+                            de.druggability[0].pharos_class : this.noValue;
+                        de.sm_druggability_display_value = de.druggability[0].sm_druggability_bucket + ': ' +
+                            de.druggability[0].classification;
+                        de.safety_rating_display_value = de.druggability[0].safety_bucket + ': ' +
+                            de.druggability[0].safety_bucket_definition;
+                        de.ab_modality_display_value = de.druggability[0].abability_bucket + ': ' +
+                            de.druggability[0].abability_bucket_definition;
+                    } else {
+                        de.pharos_class_display_value = this.noValue;
+                        de.sm_druggability_display_value = this.noValue;
+                        de.safety_rating_display_value = this.noValue;
+                        de.ab_modality_display_value = this.noValue;
+                    }
```

Comment:
Rather than trying to get sorts working with subfields, I used the displayValue approach that was used in genes-list